modules/core/shared/src/main/scala/lucuma/core/model/Proposal.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.Order._
+import cats.syntax.all._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.enum.TacCategory
+import lucuma.core.enum.ToOActivation
+import lucuma.core.math.units.IntPercent
+import monocle.Focus
+
+import scala.collection.immutable.SortedMap
+
+final case class Proposal(
+  title:         Option[NonEmptyString],
+  proposalClass: ProposalClass,
+  category:      Option[TacCategory],
+  toOActivation: ToOActivation,
+  abstrakt:      Option[NonEmptyString],
+  partnerSplits: SortedMap[Partner, IntPercent]
+)
+
+object Proposal {
```

Comment:
I wonder if we need to include the semester this proposal applies to

Reply:
And in fact we have glossed over the model changes happening from one semester to the next...

Reply:
Semester doesn't really apply equally to all proposal classes.  Some are active for less than a full semester and others over multiple semesters.  We had been discussing storing the active period for a program as start and end times.  Maybe it does makes sense to track the semester in which we expect the program to be first made active though.

Model changes should be a different problem than the one we have today.  We don't have to support opening up XMLs from any past time and continuing to migrate it up to the current version.  Instead a model change can happen whenever and we have to include a database migration at that point, and change the API and UI accordingly.  Or am I missing the point?

Reply:
Semester might be a thing to include in the `Queue` proposal class and others where it makes sense?  Though even there sometimes they'll "rollover" a queue proposal into the next semester to get it done.

Reply:
The semester thing will probably require some additional thought. Even for `LargeProgram` and `Intensive` classes, which are definitely multi-semester, the PI is supposed to be entering the `minPercentTime` that they need for this semester. They also specify the expected `totalTime` for the entire program and minimum percent that they need for that. 

So, it seems to me a proposal, at least, is only for a given semester, regardless of the lifetime of a program. 

I, for one, am not clear on how this works in Explore and the API. In Explore, do we default to the upcoming semester when creating a proposal, but give the user the option of selecting from several in the future? Do we support multiple proposals for a program? I don't recall seeing this in the mockups, but that doesn't mean it isn't there. My recollection is for a logged in user to have the ability to create a proposal if there isn't one. In the API, I have an `Option[Proposal]` on the `ProgramModel`. Maybe this needs to be a `Map[Semeter, Proposal]`?

Reply:
I guess we'll have q history of m proposal for n semesters for a give user. That maybe an issue with respect to schema evolution too

Reply:
I would think that you would always be preparing a proposal for the currently open cycle, and the only choice is what type of proposal to submit.  For example, if it's an Fast-Turnaround proposal it would be for the current month, if it's a queue proposal its for next semester, if it's a director's or poor weather proposal it's for the current semester.  It might be difficult to support proposals for times in the more distant future because we won't know which instruments will be offered or their configurations.

Reply:
Would you want to keep a record of all proposals ever? So now in 2022 you want to see a proposal sent on 2010, do we allow that?

Reply:
I hope we don't have to import old pre-GPP proposals, but going forward it seems like many programs will have associated proposal information and that will live forever along with the program.